gwlua/functions.c

```diff
@@ -355,13 +355,14 @@ static int l_setbackground( lua_State* L )
   rl_image_t* bg = **picture;
   
   int width = MAX( bg->width, 480 );
+  int x0;
   
   if ( rl_backgrnd_create( width, bg->height ) )
   {
     return luaL_error( L, "out of memory allocating the background framebuffer" );
   }
   
-  int x0 = ( width - bg->width ) / 2;
```

Comment:
Isn't it possible to tell MSVC to compile the source code as C99?

Reply:
No, for these older versions it is not, and you should just declare variables at the top of a function or code block.